docs/resources/dbaas_redisdb_replica_set.md

```diff
@@ -0,0 +1,128 @@
+---
+subcategory: "Database as a Service - RedisDB"
+layout: "ionoscloud"
+page_title: "IonosCloud: ionoscloud_redis_replicaset"
```

Comment:
on my PR this is `ionoscloud_dbaas_redis_replicaset`, which variant should we do?

 https://github.com/ionos-cloud/terraform-provider-ionoscloud/pull/576/files#diff-18a9597d085c494e8560a0946c1b8c3d75ea2f28193e65b5839156ecd842424cR4

Reply:
@avirtopeanu-ionos @cristiGuranIonos suggested `ionoscloud_redis_replicaset` so I modified using his suggestion.

Reply:
thanks!